generator/osm2meta.cpp

```diff
@@ -287,6 +287,21 @@ string MetadataTagProcessorImpl::ValidateAndFormat_wikipedia(string v) const
   return normalized;
 }
 
+
+string MetadataTagProcessorImpl::ValidateAndFormat_wikimedia_commons(string v) const
```

Comment:
Please check all possible values for this tag. They should be normalized to officially allowed values.
I've seen in taginfo simple file names, full URLs, etc.

Reply:
This is validating the tag format defined here: https://wiki.openstreetmap.org/wiki/Key:wikimedia_commons